### DISCUSSION TITLE
my scp'ed files are getting truncated at 204800 bytes after upgrading openssh

### DISCUSSION THREAD

**Discussion by tomwlane** [PowerShell/Win32-OpenSSH#2096] [General]
my scp'ed files are getting truncated at 204800 bytes after upgrading openssh to v9.2.2.0p1-Beta

I'm using an openbsd client on windows (git/bash for windows), it works on other systems, all other combinations actually, but this one system to my one upgraded system just truncates the thing last minute like.

heres the log from the client

scp: debug3: Received data 256977920 -> 257009159
scp: debug3: Short data block, re-requesting 257009160 -> 257080319 ( 1)
scp: debug3: Finish at 257080320 ( 1)
scp: debug3: Received reply T:101 I:2522 R:1
nfhl_s_fld_haz_ar.shp                                                                                                    100%  245MB  19.0MB/s   00:12
**scp: debug1: truncating at 204800**
scp: debug3: Sent message SSH2_FXP_CLOSE I:2523
scp: debug3: SSH2_FXP_STATUS 0
debug2: channel 0: read failed rfd 5 maxlen 32768: Broken pipe
debug2: channel 0: read failed

$ ls -l nfhl_s_fld_haz_ar.shp
-rw-r--r-- 1 AzureAD+TomLane 4096 204800 Jul 12 08:41 nfhl_s_fld_haz_ar.shp

and on the server
$ sshd -v
OpenSSH_9.3p1, OpenSSL 1.1.1u  30 May 2023

**Comment by maertendMSFT** [comment]
Moved to an issue here: https://github.com/PowerShell/Win32-OpenSSH/issues/2098
